### DISCUSSION TITLE
How to access Github student developer pack if my country banned all Github domains

### DISCUSSION THREAD

**Discussion by htetmyatthar** [community/community#143248] [GitHub Education] [answered]
### Hi everyone,

Hello everyone, 
I applied for the Github student developer pack with a VPN since my country(Myanmar) ISP banned [education.github](https://education.github.com/pack) this domain due to Junta and the current situation. Since I'm using VPN connection my application is rejected due to that reason since they are assuming my location is wrong. How can I get access to student developer pack and how can I access student developer pack in other ways? I'm a student who's pursuing Bachelor of Computer Science at The University of the People. I've applied for the student developer pack with my university issued email address. I've done adding my name in the BIlling information and I've added the email address to my Github account. Below is the response from the Github education team.
![image](https://github.com/user-attachments/assets/26a53d46-5641-4ac5-a6c3-7464357cc210)

**Comment by prathamlashkari** [comment]
Hi! I’m really sorry to hear about the trouble you're facing with your application for the GitHub Student Developer Pack. Since you're in Myanmar and dealing with VPN issues, here are a few suggestions that might help:

Contact GitHub Support: Explain your situation directly to GitHub support, mentioning your student status and the issues with accessing their services in Myanmar. Provide any documentation from your university to prove your enrollment.

Use University Documentation: If possible, obtain a letter or proof of enrollment from The University of the People. This can help establish your eligibility.

Check for Local Alternatives: Sometimes universities have partnerships with local organizations or other platforms that provide similar resources for students.

Explore Other Verification Methods: GitHub may offer alternative ways to verify your student status without a VPN.

Engage with the Community: Reach out to others in similar situations on forums or social media to see how they navigated this issue.

**Reply by htetmyatthar** [reply]
Hello! Thank you for your response. I'll try reaching out to the Github support. I submitted my proof of enrollment from the University of the People also. It seems that the VPN is issue.

**Reply by prathamlashkari** [reply]
If you found this helpful, please consider marking it as answered! Also, feel free to follow me on GitHub. I’d love to collaborate and build together in the future

**Comment by Bashamega** [comment]
It specifically told you not to use a VPN, which means you can use the GitHub Student Developer pack.

**Reply by htetmyatthar** [reply]
Thanks for your help. But I can't access the (https://education.github.com/discount_requests/application) link if I'm not using a VPN. That's the problem for me. I also gave the location access permission when I submitted that applicaiton. Any idea what should I do next? I made a ticket to them and hopefully I'll get a response from them.

**Reply by Bashamega** [reply]
This means that you just should not use github student pack. Or be on github

**Comment by InWamos** [comment, marked as answer]
Allow GPS access for the website from your browser and you can use VPN if you want. This should be fine. Maybe the problem was about incomplete data you provided.

P.S. Don't share your personal info here since you're asking the illegal and somebody may complain to your local feds and you may be punished for this. We are against any kind of access restrictions here btw, I'm just concerned about your safety

**Reply by htetmyatthar** [reply]
Thank you for your advice. I won't be sharing any personal information. I'll try applying with more clearer image of my admission proof.

**Reply by InWamos** [reply]
Ok, good luck and could you pls mark my message as the answer? It would me much appreciated

**Comment by htetmyatthar** [comment]
![image](https://github.com/user-attachments/assets/11563075-407a-4926-adf8-e85f0713f5b1)
Finally I got accepted to the program. Just like in the answer may be the incomplete data is the problem. I submitted with a more clear image of my admission proof and I got approved. Thanks for all your answer and I'll close this thread now.

**Reply by InWamos** [reply]
Congrats!
